.github/workflows/test.yml

```diff
@@ -51,3 +51,50 @@ jobs:
       uses: codecov/codecov-action@v1
       with:
         file: ./coverage.xml
+
+  build-win:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - uses: crazy-max/ghaction-chocolatey@v1
+    - name: Install curl and make
```

Comment:
```suggestion
    - name: Install curl and make
      uses: crazy-max/ghaction-chocolatey@v1
```

Reply:
I heve not used gitlab ci actions before. Currently, I can not understand the source of the problem. I can try to investigate it later.